pom.xml

```diff
@@ -121,7 +121,7 @@ SOFTWARE
       <plugin>
         <groupId>org.eolang</groupId>
         <artifactId>eo-maven-plugin</artifactId>
-        <version>0.28.11</version>
+        <version>0.28.10</version>
```

Comment:
@levBagryansky Why so?

Reply:
@Graur I decreased a version to avoid transitive dependencies. I get this error with previous version:
```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eolang:eo-maven-plugin:0.28.11:assemble (compile) on project eo-math: Execution compile of goal org.eolang:eo-maven-plugin:0.28.11:assemble failed: Dependency {groupId=org.eolang, artifactId=eo-sys, version=0.0.6, type=jar} contains transitive dependencies -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
```

Reply:
@levBagryansky Maybe it would be better to exclude this dependency? Did you try it?
```
            <exclusions>
              <exclusion>
                <groupId>org.eolang</groupId>
                <artifactId>eo-sys</artifactId>
              </exclusion>
            </exclusions>
```